bebraven/platform
# If Canvas email gets out of sync, set it to match Platform/Salesforce

### Description

Sync From Salesforce was failing to reconcile and fix Canvas users
who had their login email accidentally set to something different than
Platform and Salesforce. This was for two reasons:
1. We were grabbing the default notification email when returning Canvas users instead the login email. The login email is the one we care about being in sync.
2.  We weren't calling the service to check if Canvas matched Platform after determining that Salesforce and Platform were in sync

Task: https://app.asana.com/0/1174274412967132/1200197329627692

### Test Plan:
* Run Sync From Salesforce
* Sign-up the Participant using the Account Creation link (Highlander Portal Account Create Link field in SF)
* Make sure you can login
* In Canvas, as an admin, change their login email to something else so it doesn't match by looking them up here: https://braven.instructure.com/accounts/1/users and editing their "LOGIN INFORMATION"
* Then try logging in as that user. It should say the user wasn't found in Canvas (or something like that).
* Then run Sync From Salesforce.
* They should be able to login again and the login information you set in Canvas should get set back

### Types of changes




### Checklist:
